src/frontend/components/AccordionInput/AccordionInput.js

```diff
@@ -0,0 +1,19 @@
+import "./AccordionInput.css";
+export default function AccordionInputComp({ data }) {
+  return (
+    <ul>
+      {data.map((item, i) => {
+        return (
+          <li key={i} className="accordion-input">
+            <input
+              type={item.inputType}
+              id={item.typeTitle}
+              name={item.inputType}
+            />
+            <label htmlFor={item.typeTitle}>{item.typeTitle}</label>
+          </li>
+        );
+      })}
+    </ul>
```

Comment:
Mapping over the filters and passing the data as a prop to filters is really a clean way, it doesn't require repeating the same filters again and again. Awesome approach!